04.class/memo.js

```diff
@@ -27,8 +26,8 @@ class MemoApp {
   list () {
     fs.readdir('.', (err, allFiles) => {
       const files = this.#selectTextFiles(allFiles)
-      const choices = this.#collectFirstLines(files)
-      choices.forEach((element) => {
+      const texts = this.#collectFirstLines(files)
+      texts.forEach((element) => {
```

Comment:
寿命が短い変数なのでelementでも大きな問題はないですが、複数形と単数形を対応させた名前の方がよりベターかも。

```suggestion
      texts.forEach((text) => {
```